helm/alfresco-content-services/Chart.yaml

```diff
@@ -53,4 +53,8 @@ dependencies:
     version: 1.2.0
     repository: https://alfresco.github.io/alfresco-helm-charts/
     condition: alfresco-search-enterprise.enabled
+  - name: alfresco-connector-ms365
+    version: 0.1.0-alpha.0
```

Comment:
it's probably better to wait for https://github.com/Alfresco/alfresco-helm-charts/pull/62 where I am releasing the updated chart